### PR TITLE
onnx export operators added

### PR DESCRIPTION
**Pull request by Roshrini**
## Description ##
Adding operators - LogisticRegressionOutput, MakeLoss, BlockGrad, _rminus_scalar, r_div_scalar, identity

Cleaning this https://github.com/apache/incubator-mxnet/pull/12946/

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by roywei** [comment]
@mxnet-label-bot add[ONNX

**Comment by roywei** [comment]
@mxnet-label-bot add[ONNX, pr-awaiting-review]

File: tests/python-pytest/onnx/export/mxnet_export_test.py [hunk 1]
```diff
@@ -327,6 +327,73 @@ def test_softmax():
...
+            np_out = np.subtract(2, input1)
+            _test_scalar_op(input1, out, np_out)
+        if op == "Mul":
+            out = 2 * ipsym
+            np_out = np.multiply(2, input1)
+            _test_scalar_op(input1, out, np_out)
+        if op == "Div":
+            np_out = input1/2
+            out = ipsym / 2
+            _test_scalar_op(input1, out, np_out)
```
Comment (vandanavk) [inline tests/python-pytest/onnx/export/mxnet_export_test.py]: _test_scalar_op can be moved outside all the ifs

File: tests/python-pytest/onnx/export/mxnet_export_test.py [hunk 2]
```diff
@@ -327,6 +327,73 @@ def test_softmax():
...
+        if op == "rSub":
+            out = 2 - ipsym
+            np_out = np.subtract(2, input1)
+            _test_scalar_op(input1, out, np_out)
+        if op == "Mul":
+            out = 2 * ipsym
+            np_out = np.multiply(2, input1)
+            _test_scalar_op(input1, out, np_out)
+        if op == "Div":
```
Comment (vandanavk) [inline tests/python-pytest/onnx/export/mxnet_export_test.py]: is there a different test case for rdiv scalar?

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 3]
```diff
@@ -975,6 +1023,13 @@ def convert_div_scalar(node, **kwargs):
     """
     return scalar_op_helper(node, 'Div', **kwargs)
 
+@mx_op.register("_rdiv_scalar")
+def convert_rdiv_scalar(node, **kwargs):
+    """Map MXNet's _rdiv_scalar operator attributes to onnx's Div operator.
+    Creates a new node for the input scalar value, adds it to the initializer
+    and return multiple created nodes.
+    """
+    return scalar_op_helper(node, 'Div', **kwargs)
```
Comment (vandanavk) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py]: could you add _power_scalar too and the relevant test case.
once this is added tests with *bcast* can be enabled

**Comment by Roshrini** [comment]
Created new PR - https://github.com/apache/incubator-mxnet/pull/13821

**Pull request closed without merging**